zScanner/ViewControllers/NewDocument/NewDocumentPhotosViewController.swift

```diff
@@ -212,9 +212,8 @@ class NewDocumentPhotosViewController: BaseViewController {
 extension NewDocumentPhotosViewController: UIImagePickerControllerDelegate, UINavigationControllerDelegate {
     func imagePickerController(_ picker: UIImagePickerController, didFinishPickingMediaWithInfo info: [UIImagePickerController.InfoKey : Any]) {
         if let pickedImage = info[.originalImage] as? UIImage {
-            #warning("TODO Am I do it right here?")
-            let pickedPicture = PageDomainModel(image: pickedImage, index: 0, correlationId: "")
```

Comment:
This was not correct. You are filling up index 0 and empty correlation ID which are important later for correct upload of the picture. For this reason, it was not a good decision to create PageDomainModel here. I have separated Page and PageDomainModel. Page is JUST image and description, do correlation ID and index can be added later as it was before.